lib/config/options/index.ts

```diff
@@ -2030,6 +2030,16 @@ const options: RenovateOptions[] = [
     type: 'boolean',
     default: true,
   },
+  {
+    name: 'gitUrl',
+    description:
+      'Overrides the default resolution for git remote, e.g. to switch GitLab from HTTPS to SSH-based. Currently GitLab only. Can be one of `default`, `endpoint` or `ssh`',
```

Comment:
```suggestion
      'Overrides the default resolution for git remote, e.g. to switch GitLab from HTTPS to SSH-based.',
```

Trim, as it will be merged with the markdown docs.